### DISCUSSION TITLE
Can't remove the forked repository from my account

### DISCUSSION THREAD

**Discussion by GoldExynos** [community/community#32808] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by airtower-luna** [comment, marked as answer]
That sounds like something to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Comment by A-Unix** [comment]
If you still got no solution, then here's the one:

1. Open the repo you want to detach.
2. Click on the settings(as shown in image 1)
3. Scroll down to the end of the page and click on the 'Delete this repository' option(See image 2)
4. Just follow the next instructions and it will be removed from your repositories.

Image 1: ![image](https://github.com/community/community/assets/54507991/91da4869-a2d1-4559-aac3-c2f51714db94)

Image 2: ![image](https://github.com/community/community/assets/54507991/6597a9ab-8ecf-422f-a8de-0991c16e3f6c)

**I hope this info helps solve your query.**
